### ISSUE TITLE
FR: Add option to scaffold new projects without extra dependencies or boilerplate

### ISSUE THREAD

**Issue by anastasiya29** [Sitecore/jss#143]
When a new project is scaffolded via `jss create <project-name> <framework>`, there are 2 things that a developer must do to clean up the solution to get it to a skeleton that doesn't have any boilerplate or extra dependencies.
1. Follow the instructions from localhost:3000 about which Styleguide and GraphQL files to delete
2. Look through package.json and figure out which dependencies can be removed (like bootstrap and axios)

The boilerplate and extra dependencies are great as a sample for someone who is learning, but they are not needed every time. It would be good to have an option (a flag in the `jss create` command) to create a "bare bones" solution.

**Comment by 3stacks** [comment]
Agreed. Removing graphql has caused issues for the Experience Editor for me
